draft-ietf-tls-tls13.md

```diff
@@ -1742,24 +1744,24 @@ Here:
 The list of extension types is maintained by IANA as described in
 {{iana-considerations}}.
 
-The client sends its extensions in the ClientHello. The server MAY
-send extensions in the ServerHello, EncryptedExtensions, Certificate,
-and HelloRetryRequest messages. The NewSessionTicket also allows
-the server to send extensions to the client though these are
-not directly associated with the extensions in the ClientHello.
+Extensions are generally structured in a request/response fashion (though
+some extensions are just indications with no corresponding response). The client
+sends its "request" extensions in the ClientHello message and the server sends
+its response extensions in the ServerHello, EncryptedExtensions
```

Comment:
I find the construct "request" extension and "response" extension awkward.  How about extension request and extension reponse, and if an extension is only a signalling thing, then just plain extzension.